File: CodingBat/Python/Warmup_1/monkey_trouble.py
#We have two monkeys, a and b, and the parameters a_smile and b_smile indicate if each is smiling. We are in trouble if they are both smiling or if nither of them is smiling. Return True if we are in trouble.
def monkey_trouble(a_smile, b_smile):
    if (a_smile and b_smile) or (not a_smile and not b_smile):
        return True
    return False
print monkey_trouble(False,False)    

File: CodingBat/Java/Warmup_1/monkeyTrouble.java
public class monkeyTrouble{
    // We have two monkeys, a and b, and the parameters aSmile and bSmile indicate if each is smiling. we are in trouble if they are both smiling or if none of them is smiling. Return true if we are in trouble.
    public static boolean monkeytrouble(boolean aSmile, boolean bSmile){
        if ((aSmile && bSmile)|| (!aSmile && !bSmile)){
            return true;
        }
        return false;
    }
    public static void main(String[] args) {
        
    }
}